panacea/web/static/css/app.css

```diff
@@ -34,3 +34,7 @@
   height: 128px;
   margin: 40px;
 }
+
+.padded-bottom {
```

Comment:
Instead of this padded class will using bootstrap `row` bits get us the right padding we want?